### ISSUE TITLE
Dataset to be used for training

### ISSUE THREAD

**Issue by guptasonam1602** [Raj-08/tensorflow-object-contour-detection#14]
Hi, 
Thanks for sharing the code. I am little unsure about the dataset that shoud be used for training. Also, how it should be pre-processed. The paper says that PASCAL VOC 2012 has been used with some CRF pre-processing on the polygon annotations provided for the segmentation. 

 I have downloaded PASCAL VOC 2012 from http://host.robots.ox.ac.uk/pascal/VOC/voc2012/#data . The images folder contains 17000 images. How should I generate the ground truth from this?

Hoping to get some response. 

Thanks

**Comment by rubeea** [comment]
The ground-truth is already within the PASCAL VOC dataset. Go to VOC2012> Segmentation Class folder. You can see all the ground-truth masks over there.

**Comment by madhumilan** [comment]
> The ground-truth is already within the PASCAL VOC dataset. Go to VOC2012> Segmentation Class folder. You can see all the ground-truth masks over there.

Can you please tell me how did you exactly train the model? Even I tried using the Segmentation Class folder as the ground truth values.
